tests/subsys/bluetooth/mesh/scheduler_model/message_validity/src/main.c

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <zephyr/ztest.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/timeutil.h>
+#include <bluetooth/mesh/gen_onoff_srv.h>
+#include <bluetooth/mesh/time_srv.h>
+#include <bluetooth/mesh/scheduler_srv.h>
+#include <bluetooth/mesh/scheduler_cli.h>
+#include <scheduler_internal.h>
+#include <model_utils.h>
+#include <time_util.h>
+#include <sched_test.h>
+
+static struct tm start_tm;
+
+struct bt_mesh_scheduler_cli sched_cli;
```

Comment:
I didn't find where scheduler client is used.

Reply:
It is used in the `bt_mesh_scheduler_cli_action_get` call. It is also listed as a model in the mock element.